.pre-commit-config.yaml

```diff
@@ -52,8 +52,8 @@ repos:
     rev: v1.49.0
     hooks:
     - id: golangci-lint
-      entry: bash -c 'cd go/adbc && golangci-lint run --fix'
-      args: [--fix]
+      entry: bash -c 'cd go/adbc && golangci-lint run --fix --timeout 2m'
```

Comment:
we're hitting the timeout??

Reply:
The default timeout is 1m, and i can't reproduce hitting the timeout locally. The only thing I can think of is that behind the scenes it's grabbing the dependencies or something and *that's* taking longer? I don't know. It's really strange and I haven't been able to reproduce it anywhere but in the GHA runner....

Reply:
Well, I'd believe that the runner is just slower than our local machines so that's OK.